src/python/turicreate/__init__.py

```diff
@@ -19,9 +19,11 @@
 __version__ = '{{VERSION_STRING}}'
 from turicreate.version_info import __version__
 
-import turicreate.toolkits
 from turicreate.toolkits import evaluation
-import turicreate.aggregate
+# expose name aggregate as global variable
+# this won't interfere with import turicreate.aggregate as agg
```

Comment:
Does it interfere with tab complete in IPython?

Reply:
nope.